### ISSUE TITLE
I don't know what title should I give, pls read this

### ISSUE THREAD

**Issue by venkat2305** [Frontesque/VueTube#237]
When we play a video , app downloads the next few minutes or seconds gradually. Can you do something so that , if a SponsorBlock segment is present, that portion of the video won't be loaded while streaming. we can save some data.
What do you say?

**Comment by MySurmise** [comment]
I disagree. It should still be loaded, look at how it works with the browser extension: When the user presses enter, it still plays that portion of the video.

**Comment by venkat2305** [comment]
oh,  i personally skip all the segments. anyhow it doesn't matter now.
